.circleci/config.yml

```diff
@@ -3,8 +3,10 @@ orbs:
   aws-s3: circleci/aws-s3@2.0.0
 parameters:
   cross-container-tag:
+    # when updating the go version, should also update the go version in go.mod
```

Comment:
I still don't understand how this tag is derived. obviously we have the go version, but what repo/branch does the sha come from? Is the tag a copy/paste from anywhere or is it wholly constructed?

The answers to these questions was what I had envisioned going into the comment.

Reply:
The tag is created here: https://github.com/influxdata/edge/blob/master/.circleci/config.yml#L56

It is the go version and the exact commit of the edge repo that the container was built from.